src/shared/bus-util.c

```diff
@@ -130,6 +130,7 @@ int bus_event_loop_with_idle(
                         /* Inform the service manager that we are going down, so that it will queue all
                          * further start requests, instead of assuming we are already running. */
                         sd_notify(false, "STOPPING=1");
+                        sd_notify_barrier(0, 5 * USEC_PER_SEC);
```

Comment:
- Please use boolean with short comment: `0` -> `/* unset_environment = */ false`
- Maybe, better to log at least when `-ETIMEDOUT` ? See notify/notify.c.